test/integration/test_client_easy.c

```diff
@@ -0,0 +1,116 @@
+#include "../../include/dqlite.h"
+#include "../lib/munit.h"
+#include "../lib/runner.h"
+
+#include <dirent.h>
+#include <unistd.h>
+
+SUITE(client_easy);
+
+#define N_SERVERS 3
+
+struct fixture
+{
+	char *dirs[N_SERVERS];
+	dqlite_server *servers[N_SERVERS];
+};
+
+static void *setup(const MunitParameter params[], void *user_data)
+{
+	(void)params;
+	(void)user_data;
+	struct fixture *f = munit_malloc(sizeof *f);
+	unsigned i;
+	char dirpath[100];
+	char *p;
+	int rv;
+
+	for (i = 0; i < N_SERVERS; i += 1) {
+		strncpy(dirpath, "/tmp/dqlitetestXXXXXX", 100);
+		p = mkdtemp(dirpath);
+		if (p == NULL) {
+			munit_error("mkdtemp");
+		}
+		f->dirs[i] = strdup(dirpath);
```

Comment:
There is a `test_dir_setup()` helper declared in `test/lib/fs.h`, it will reduce boilerplate/duplication, e.g.:

```C
f->dirs[i] = test_dir_setup();
rv = dqlite_server_create(f->dirs[i], &f->servers[i]);
```